### PR TITLE
UIREQ-834: TLR: Inconsistent title info loading when request created from Instance record

### PR DESCRIPTION
**Pull request by artem-blazhko**
## Purpose
When the user tries to create a request from Inventory item information is not loaded the first time. But when the user refreshes the page everything works as expected.

## Refs
[UIREQ-834](https://issues.folio.org/browse/UIREQ-834)

### CODE REVIEW

File: src/routes/RequestsRoute.js [hunk 1]
```diff
@@ -402,7 +405,9 @@ class RequestsRoute extends React.Component {
       locale: PropTypes.string.isRequired,
     }).isRequired,
     history: PropTypes.object,
-    location: PropTypes.object.isRequired,
+    location: PropTypes.shape({
+      search: PropTypes.string,
+    }).isRequired,
```
Comment (zburke) [inline src/routes/RequestsRoute.js:410]: Thanks for providing this more precise description of the shape of location. `PropTypes.object` borders on useless 🙄 . If you're going to do this, however, the new definition has to be complete: `pathname` is missing and now causes a lint error on line 1033.

**Review by zburke — CHANGES_REQUESTED** [review]
Looks good but we need to clean up the newly introduced lint error.

File: src/routes/RequestsRoute.js (hunk 1 above)
Comment (artem-blazhko) [reply src/routes/RequestsRoute.js]: Thank you @zburke. Fixed

**Comment by artem-blazhko** [comment]
Hello @zburke. Could you check it one more time?

**Review by zburke — APPROVED** [review]
LGTM; thank you! 🤝

**Pull request merged**